boms/cloud-oss-bom/pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>24.4.1-SNAPSHOT</version>
+  <version>25.0.1-SNAPSHOT</version>
```

Comment:
Why a major version bump?

Reply:
The version is 25.0.0 (prev: 24.4.0) because there are major version bumps in google-cloud-dataproc and google-cloud-service-management (https://github.com/googleapis/java-cloud-bom/releases/tag/v0.170.0).

Reply:
Why isn't `java-cloud-bom` bumped to the next major version then?

Reply:
java-cloud-bom versioning is different from the Libraries BOM.

Reply:
It's not semver?

Reply:
Google Cloud BOM is semver but marked as pre-1.0. It always bumps the minor version.